src/example-types/text.js

```diff
@@ -37,7 +47,7 @@ module.exports = {
       return result
     }
 
-    if (lookAtUntouched) fieldName += '.lowercased'
+    if (lookAtUntouched) fieldName += '.untouched'
```

Comment:
If this is actually more performant, that's awesome because we don't need another analyzer.

We should additionally expose an option to configure case sensitivity.